### DISCUSSION TITLE
Add filters + sorting instead of algorithmic "For you" feed

### DISCUSSION THREAD

**Discussion by SMUsamaShah** [community/community#14240] [Other Feature Feedback, Questions, & Ideas]
Need a feed I can control myself. For people who prefer AI generated feed, provide that as an option (see Kickstarter 'Sort by Magic' feed) for example.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Comment by SMUsamaShah** [comment]
These options could have been just added in the "Following" feed. :|
Went through each of the options.
Why is "Releases" showing people I am not following and have no obvious connection with?

**Reply by kellytk** [reply]
When it states "Recommended based on people you follow" and I follow noone, I know it's the same Microsoft (:hankey:) it's always been.

**Reply by jfullstackdev** [reply]
it's being fixed by now there was one reply in another thread, it's related to seeing repos that you already starred
